src/organisms/cards/SelectCard/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import CondensedSelectField from 'molecules/formfields/Select/CondensedSelectField';
+import styles from './select-card.module.scss';
+
+
+function SelectCard ( props ) {
+  const {
+    className,
+    label,
+    placeholder,
+    selectOptions,
+    footerText,
+    linkUrl,
+    onClick,
+    onChange,
+    value
+  } = props;
+
+  return (
+    <div className={ classnames( styles.select, className ) }>
+      <CondensedSelectField
+        label={ label }
+        placeholder={ placeholder }
+        selectOptions={ selectOptions }
+        onChange={ onChange }
+        value={ value }
+      />
+
+      <footer className={ styles.footer }>
+        <div className={ styles['footer-link-wrapper'] }>
+          <a
```

Comment:
you might need to rebase but we have a link component now! also I wonder if eventually this footer could become a component?